.github/workflows/run-tests-on-new-version-of-ddc-sc.yml

```diff
@@ -35,5 +35,16 @@ jobs:
         run: |
           echo "DDC_SMART_CONTRACT_VERSION::$DDC_SMART_CONTRACT_VERSION"
           docker build --build-arg ECR_REGISTRY=$ECR_REGISTRY --build-arg DDC_SMART_CONTRACT_VERSION=$DDC_SMART_CONTRACT_VERSION -f Dockerfile.tests -t pos-network-node:test .
+      - name: Slack Notification
+        if: ${{ always() }}
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+          SLACK_CHANNEL: e2e-test-results
+          SLACK_COLOR: ${{ job.status }}
+          SLACK_MESSAGE: 'Run tests with params:
```

Comment:
Could we use different messages depending on tests status? It will be great if we can use mentions for red results.